domain/src/test/java/com/distillery/android/domain/FakeToDoRepositoryTest.kt

```diff
@@ -24,7 +25,8 @@ class FakeToDoRepositoryTest {
 
     @Before
     fun setup() {
-        repository = FakeToDoRepository()
+        val scope = GlobalScope
+        repository = FakeToDoRepository(scope)
```

Comment:
Minor: can be inlined
```suggestion
        val scope = GlobalScope
        repository = FakeToDoRepository(scope = GlobalScope)
```